index.html

```diff
@@ -551,6 +551,68 @@ <h2>Exposing MediaStreamTrack source background blur support</h2>
 };</pre>
     </div>
   </section>
+  <section>
+    <h2>Exposing MediaStreamTrack source eye gaze correction support</h2>
+    <p>Some platforms or User Agents may provide built-in support for human eye
+       gaze correction, in particular for camera video streams.
```

Comment:
Should we clarify what gaze correction is? E.g. "when eye gaze correction is enabled, the User Agent will modify the video to make the eyes of faces appear to look at the camera"